packages/core/src/2d/text/Font.ts

```diff
@@ -0,0 +1,50 @@
+import { RefObject } from "../../asset/RefObject"
+import { assignmentClone } from "../../clone/CloneManager";
+import { Engine } from "../../Engine";
+import { UpdateFlag } from "../../UpdateFlag";
+import { UpdateFlagManager } from "../../UpdateFlagManager";
+
+export class Font extends RefObject {
+  @assignmentClone
+  private _fontName: string = "Arial";
+
+  private _updateFlagManager: UpdateFlagManager = new UpdateFlagManager();
```

Comment:
Maybe add `@ignoreClone`

Reply:
should not need, has handle in TextRenderer like this:

```
/**
 * @internal
 */
_cloneTo(target: TextRenderer): void {
  target.font = this._font;
}
```